helm/messenger/conf/nginx.conf

```diff
@@ -73,6 +73,14 @@ server {
     expires    1m;
   }
 
+  location = /terms {
+    try_files /assets/assets/terms.html =404;
```

Comment:
@tyranron, this redirects to `/var/www/public/assets/assets/terms.html` file, since `root` is defined higher as `/var/www/public`. This directory, `assets`, is embedded by Flutter when building application for web.

Reply:
@tyranron, this is the files structure Flutter builds for web: <img width="370" alt="Снимок экрана 2024-04-19 в 17 05 36" src="https://github.com/team113/messenger/assets/21217248/0d39d02c-5069-497d-ba79-c122614292bd">

Reply:
Flutter embeds the assets provided in the `pubspec.yaml` into the `assets` directory. And since our assets directory is called `assets`, that creates two `assets` directories.

Reply:
This is URI, not a path. OK.